great_expectations/data_context/data_context/context_factory.py

```diff
@@ -140,6 +141,57 @@ def get_validator(self, batch_request: BatchRequest) -> Validator:
             )
         return self._project.get_validator(batch_request=batch_request)
 
+    def is_using_cloud(self) -> bool:
+        if not self._project:
+            raise RuntimeError(
+                "This action requires an active DataContext. "
+                + "Please call `great_expectations.get_context()` first, then try your action again."
+            )
+
+        from great_expectations.data_context import CloudDataContext
```

Comment:
Can we move this import up?

Reply:
Needs to be inline but can move to top of the func